thorlcr/activities/thdiskbaseslave.cpp

```diff
@@ -299,6 +299,13 @@ void CDiskReadSlaveActivityBase::serializeStats(MemoryBuffer &mb)
         stats.merge(activeStats);
     }
     stats.setStatistic(StNumDiskRowsRead, diskProgress);
+    if (!isEmptyString(logicalFilename))
+    {
+        unsigned __int64 numDiskReads = activeStats.getStatisticValue(StNumDiskReads);
+        StatsScopeId scope(SSTfile, logicalFilename);
```

Comment:
It's less of a concern if these are only serialized when done (because only used at moment when graph is done).
However, volume of stats could still be an issue, if these acts are in rapidly repeatedly executing child queries.

Reply:
@jakesmith how do superfiles fit in to this?